config.nix

```diff
@@ -0,0 +1,101 @@
+{
+ # extend the shell with buildInputs specific to this project
+ buildInputs = [ ];
+
+ # configure holonix itself
+ holonix = {
+
+  # true = use a github repository as the holonix base (recommended)
+  # false = use a local copy of holonix (useful for debugging)
+  use-github = true;
+
+  # configure the remote holonix github when use-github = true
+  github = {
+
+   # can be any github ref
+   # branch, tag, commit, etc.
+   ref = "v0.0.84";
+
+   # the sha of what is downloaded from the above ref
+   # note: even if you change the above ref it will not be redownloaded until
+   #       the sha here changes (the sha is the cache key for downloads)
+   # note: to get a new sha, get nix to try and download a bad sha
+   #       it will complain and tell you the right sha
+   sha256 = "1hlp4khar48ab48pnvjcwhkmrgpznxwlnw8dk24iw844sskrzv7z";
+
+   # the github owner of the holonix repo
+   owner = "holochain";
+
+   # the name of the holonix repo
+   repo = "holonix";
+  };
+
+  # configuration for when use-github = false
+  local = {
+   # the path to the local holonix copy
+   path = ./.;
+  };
+
+ };
+
+ # configure the release process
+ release = {
+  hook = {
+   # sanity checks before deploying
+   # to stop the release
+   # exit 1
+   preflight = ''
+hn-release-hook-preflight-manual
+'';
+
+   # bump versions in the repo
+   version = ''
+'';
+
+   # publish artifacts to the world
+   publish = ''
+echo "All finished!!!"
+'';
+  };
+
+  # the commit hash that the release process should target
+  # this will always be behind what ends up being deployed
+  # the release process needs to add some commits for changelog etc.
+  commit = "926efebf26d69ee4d6cb9ac2d78fe04531e1458b";
+
+  # the semver for prev and current releases
+  # the previous version will be scanned/bumped by release scripts
+  # the current version is what the release scripts bump *to*
+  version = {
+   current = "0.0.1-dev.9";
+   previous = "_";
+  };
```

Comment:
Hmm I really don't want to involve nix in the simple process of cutting a new npm release. Raw npm is perfectly capable of this. @thedavidmeister we talked about using holonix less for ancillary repos like this, would this be a good time to break from the mold?

Reply:
Ah, I'm not opinionated on how releases should happen. Can leave that part blank, and just not use `hn-release-github` etc.